diff --git a/node_modules/react-native-web/dist/index.js b/node_modules/react-native-web/dist/index.js
index 4f69987..df96e50 100644
--- a/node_modules/react-native-web/dist/index.js
+++ b/node_modules/react-native-web/dist/index.js
@@ -67,3 +67,17 @@ export { default as DeviceEventEmitter } from './exports/DeviceEventEmitter';
 export { default as useColorScheme } from './exports/useColorScheme';
 export { default as useLocaleContext } from './exports/useLocaleContext';
 export { default as useWindowDimensions } from './exports/useWindowDimensions';
+
+import Animated from './exports/Animated';
+import { useRef } from 'react';
+
+export function useAnimatedValue(
+    initialValue,
+    config,
+) {
+    const ref = useRef(null);
+    if (ref.current == null) {
+        ref.current = new Animated.Value(initialValue, config);
+    }
+    return ref.current;
+}
