diff --git a/node_modules/expo-file-system/src/next/FileSystem.ts b/node_modules/expo-file-system/src/next/FileSystem.ts
index b020328..473c0fe 100644
--- a/node_modules/expo-file-system/src/next/FileSystem.ts
+++ b/node_modules/expo-file-system/src/next/FileSystem.ts
@@ -74,7 +74,7 @@ export class FileBlob extends Blob {
   }
 
   slice(start?: number, end?: number, contentType?: string): Blob {
-    return new Blob([this.file.bytes().slice(start, end)], { type: contentType });
+    return new Blob([this.file.bytes().slice(start, end)], { type: contentType! });
   }
 }
 
